index.html

```diff
@@ -1,13 +1,50 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="manifest" href="/manifest.json" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- pwa icons -->
+    <link rel="icon" href="icons/android-chrome-192x192.png" type="image/png" />
+    <link rel="alternate icon" href="/favicon.ico" type="ico" sizes="16x16" />
+    <link rel="apple-touch-icon" href="icons/apple-touch-icon.png" sizes="180x180" />
+
+    <!-- og -->
+    <meta name="title" property="og:title" content="MCY | 목천교회 청년부" />
+    <meta
+      name="description"
+      property="og:description"
+      content="하나님 안에서 세워나가는 목천교회 청년부의 다양한 콘텐츠를 확인하고, 청년들을 위해 기도는 믿음의 공동체가 되길 바랍니다.
```

Comment:
```suggestion
      content="하나님 안에서 세워나가는 목천교회 청년부의 다양한 콘텐츠를 확인하고, 청년들을 위해 기도는 믿음의 공동체가 되길 바랍니다.
```
이거 문구 이상함 ㅋㅋㅋ 간결하게 바꿔봐! 읽어보면 상당히 이상함